docs/tutorials/projects/helmet.md

```diff
@@ -425,20 +427,38 @@ Configure a trigger as follows:
    ```json {class="line-numbers linkable-line-numbers"}
      "triggers": [
        {
-         "url": "<Insert your own cloud function URL>",
          "event": {
            "attributes": {
              "data_types": ["binary"]
            },
            "type": "part_data_ingested"
+         },
+         "notifications": [
+         {
+          "type": "webhook",
+          "value": "<Insert your own cloud function URL>",
+          "seconds_between_notifications": 60
          }
-       }
-     ]
+        ]
+      }
+   ]
    ```
 
-3. Replace the `url` value with your cloud function URL.
+3. Replace the `value` in `notifications` with your cloud function URL.
    You can get this URL by copying it from the **TRIGGER** tab in the cloud function console.
-   Once you've done this, the `url` line should resemble, for example, `"url": "https://us-east1-example-string-123456.cloudfunctions.net/hat-email"`.
+   Once you've done this, the `value` line should resemble, for example, `"value": "https://us-east1-example-string-123456.cloudfunctions.net/hat-email"`.
+
+   If you skipped writing a custom cloud function in favor of the built-in email functionality, use this JSON for the `notifications` object instead:
```

Comment:
I think put this in an expander and at the top of this segment or something like that. If people do the generic email way, they shouldn't have to read through all the rest

Reply:
I ended up putting both in expanders for maximum cleanness, LMK what you think!

Reply:
and including the full snippets both times